docs/basic-features/typescript.md

```diff
@@ -118,3 +118,17 @@ export default (req: NextApiRequest, res: NextApiResponse<Data>) => {
   res.status(200).json({ name: 'John Doe' })
 }
 ```
+
+## Custom `App` using `_app.tsx`
```

Comment:
I'll prefer either a matching title with the other section, like the other titles in this section, or to be even more specific with:
```
## Custom `App` using `pages/_app.tsx
```

```suggestion
## Custom `App`
```